pybamm/models/standard_variables.py

```diff
@@ -312,7 +313,7 @@ def __init__(self):
         )
 
     def __setattr__(self, name, value):
-        value.print_name = name
+        value.print_name = prettify_print_name(name)
```

Comment:
we could make `print_name` a property in `symbol` and then have the setter class for the property automatically call `prettify_print_name`